### DISCUSSION TITLE
this shit broken

### DISCUSSION THREAD

**Discussion by rylanm2002** [3kh0/ext-remover#306] [General JOIN TITANIUM NETWORK TO SHARE PLEASE]
bruh this not workin anymore or at least on windows its not

**Comment by rwg100** [comment]
im pretty sure it was only made to work on chromebooks

also on windows computers, the system uses an app instead of an extension so you cant just remove and extension.

**Comment by bartholemuew** [comment]
bro gotta be stupid 💀

**Comment by obkduishinijf** [comment]
shut the fuck up

**Reply by fnfgo** [reply]
Take your own advice kid

**Reply by obkduishinijf** [reply]
> Take your own advice kid

bro who are you 💀

**Reply by Theblackmamba824** [reply]
💀

**Reply by jazzymakesstuff** [reply]
ive never seen either of you 😭

**Comment by Aka-but-nice** [comment]
BRO, you complain it does not work for windows when it is for chromeOS, shut your tangerine mr clean can i get a jelly bean urine stain rusty drain destiny 2 dlc campaign builder hut ingrown nut spread me wide fill me up flint and steel banana peel dream just did a face reveal looking ass out of my face bill from bill and mandy special needs vigilante hair cut cost you 4 rare candies dusty divot disabled cricket e-dating a minecraft piglet poverty stricken dating subscription severe addiction to popeyes chicken josh and drake polluted lake for your birthday your mama bought you a urinal cake "yeah thanks mom" stop playing bruh you stupid omnisexual banana chip looking ass

this dude brought out the whole dictionary for this

**Reply by 3kh0** [reply]
heck ya!

**Reply by fnfgo** [reply]
Bro fr became a rapper

**Reply by Aka-but-nice** [reply]
Fr

**Reply by Aka-but-nice** [reply]
Who thought a Maintainer, would have bars?

**Reply by NathanielThaBeast** [reply]
Its from packgod on youtube

**Reply by Aka-but-nice** [reply]
fr

**Comment by Aka-but-nice** [comment]
> bruh this not workin anymore or at least on windows its not


It's not working on chrome os either, that is basically at this point the whole reason for this discussion, also let's be honest until pingot comes out nothing ain't working the only thing we can rely on is the method that revolves around, using the chrome://signOut method, and FYI it's only for Chromebooks!

**Reply by rylanm2002** [reply]
ur the only person here with a brain

**Reply by Theblackmamba824** [reply]
💀

**Reply by piplup7575** [reply]
ong just downgrade
https://chrome100.dev

**Reply by fnfgo** [reply]
> ong just downgrade https://chrome100.dev

you could get arrested for that you know.
But i dont care. I want my freedom back

**Reply by rwg100** [reply]
thats literally illegal

**Reply by fnfgo** [reply]
> I dont think he cares

i dont :)

**Reply by ghost** [reply]
how is it illegal

**Reply by jazzymakesstuff** [reply]
nobody catches you if you pirate so nobody is going to catch you if you downgrade (no school district really cares about your chrome version because most IT techs dont look into it).

**Comment by Zestiko** [comment]
Чт, 29 дек. 2022 г. в 11:04, Mahi Muhammad ***@***.***>:

> Hi
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/306#discussioncomment-4542486>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJOLGHWB2Z4AV2OUIIU5LCTWPVHYFANCNFSM6AAAAAASYF4P5U>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
Hello

**Comment by ghost** [comment]
lollll

**Comment by fnfgo** [comment]
I see

**Reply by S-PScripts** [reply]
you came back out of nowhere
